doc/045_working_with_repos.rst

```diff
@@ -82,6 +82,64 @@ Furthermore you can group the output by the same filters (host, paths, tags):
     1 snapshots
 
 
+Copying snapshots between repositories
+======================================
+
+In case you want to transfer snapshots between two repositories, for
+example from a local to a remote repository, you can use the ``copy`` command:
+
+.. code-block:: console
+
+    $ restic -r /srv/restic-repo copy --repo2 /srv/restic-repo-copy
+    repository d6504c63 opened successfully, password is correct
+    repository 3dd0878c opened successfully, password is correct
+
+    snapshot 410b18a2 of [/home/user/work] at 2020-06-09 23:15:57.305305 +0200 CEST)
+      copy started, this may take a while...
+    snapshot 7a746a07 saved
+
+    snapshot 4e5d5487 of [/home/user/work] at 2020-05-01 22:44:07.012113 +0200 CEST)
+    skipping snapshot 4e5d5487, was already copied to snapshot 50eb62b7
+
+The example command copies all snapshots from the source repository
+``/srv/restic-repo`` to the destination repository ``/srv/restic-repo-copy``.
+Snapshots which have previously been copied between repositories will
+be skipped by later copy runs.
+
+Note that this process will have to read (download) and write (upload) the
+entire snapshot(s) due to the different encryption keys used on the source
+and destination repository.
```

Comment:
I suggest changing this to the following (just to make it stand out in a note box):

~~~
.. note:: Note that this process will have to read (download) and write (upload) the
    entire snapshot(s) due to the different encryption keys used in the source and
    destination repository. Also, the transferred files are not re-chunked, which
    may break deduplication between files already stored in the destination repo
    and files copied there using this command.
~~~

Reply:
Is changed.